### ISSUE TITLE
unable to verify phone number on signal android while using screen reader

### ISSUE THREAD

**Issue by crentava** [signalapp/Signal-Android#11540]
Let's begin with a checklist: Replace the empty checkboxes [ ] below with checked ones [x] accordingly. -->

- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.
when I attempt to verify my phone number, I get a verification code, but I am unable to enter said code due to lack of a textfield
### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug
- install a fresh copy of signal
- turn on the screen leader that is included with android by going to accessibility settings/talkback and toggle the switch to turn it on
- open the freshly installed signal and enter the number that you want to verify
- wait to receive your code
**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
you'll be taken to the code verification screen, but you won't have a place to enter your code
**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)
you should be able to find a text field labeled "enter the code you received" or something similar.
### Device info

**Device:** Manufacturer Model XVI
Samsung galaxy S8
**Android version:** android 9
**Signal version:** 0.0.0
version 5.19.4
### Link to debug log

https://debuglogs.org/6a959289be19af7fc05f3f6913dbcc842a274bc76bc83e066e0e2c616f7c3c67.txt
